### DISCUSSION TITLE
Using bal:condition in bundle element fails with errors

### DISCUSSION THREAD

**Discussion by AndreFo** [wixtoolset/issues#7197] [Questions] [answered]
Using wixtoolset version 4.0.0-rc.2 and Heatwave in VS2022 17.4.4 I created an Msi Project with name MyPackage and a Bundle Project which references the msi project. I added a condition to Bundle.wxs:
```
<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
  <Bundle Name="BundleTestCondition" Manufacturer="TODO Manufacturer" Version="1.0.0.0" UpgradeCode="5a05585f-e843-4828-872d-1266930f714f">
    <BootstrapperApplication>
      <bal:WixStandardBootstrapperApplication LicenseUrl="https://www.example.com/license" Theme="hyperlinkLicense" />
    </BootstrapperApplication>

    <Chain>
      <MsiPackage SourceFile="$(var.MyPackage.TargetPath)" />
    </Chain>

    <bal:Condition Message="MyPackage requires 64-bit Windows.">VersionNT64</bal:Condition>

  </Bundle>
</Wix>

```

The build generates the following error: 

```
BundleTestCondition\BundleTestCondition\Bundle.wxs(11): error WIX0400: The Condition element contains illegal inner text: 'VersionNT64'.
BundleTestCondition\BundleTestCondition\Bundle.wxs(11): error WIX0020: The Condition element's inner text cannot be an empty string or completely whitespace. If you don't want a condition, then simply remove the entire Condition element.
```
Any idea why the inner text for the condition is considered empty or whitespace?

**Comment by barnson** [comment, marked as answer]
Use the `Condition` attribute: https://wixtoolset.org/docs/reference/schema/bal/condition/

**Comment by AndreFo** [comment]
Thanks, this is working. The docs still say that 

> The condition is specified in the inner text of the element.

However, it indicates that Condition is an attribute. I have overlooked that.

**Reply by barnson** [reply]
Every page has a `[Edit the schema for this page](https://github.com/wixtoolset/web/blob/master/src/xsd4/bal.xsd)` link for making corrections like that.
